torch/onnx/_internal/exporter.py

```diff
@@ -221,14 +289,13 @@ def __init__(
             self.options = options
         assert self.options is not None
 
-        self.model: torch.nn.Module = (
```

Comment:
I'm not sure if this is needed. For this I could revert.

At least the pytree flatten part inside `WrappedFuncModule` is not needed anymore.